Resources/public/javascript/sticky.js

```diff
@@ -0,0 +1,34 @@
+require('@wearejust/sticky');
+
+let addBtnClone;
+let elOffsetLeft;
+let $window = $(window);
+let minWidth = 940;
+
+$(function() {
+	if($window.width() > minWidth) {
+		$('.add_records').attr('data-sticky-id', 'sticky_add_btn');
+		elOffsetLeft = $('.add_records').offset().left;
+		addBtnClone = $('.add_records').clone().appendTo('.navbar-collapse');
+		addBtnClone.addClass('js-sticky');
+		addBtnClone.css('left', elOffsetLeft);
+		$(addBtnClone).attr('data-sticky-target', 'sticky_add_btn');
+	    $('.js-sticky').sticky({
+	        active: 'is_sticky',
+	        topHeightOffset: -1.1
+	    });
+	    $window.resize();
+	}
+});
+
+$window.resize(function() { 
+	if($window.width() > minWidth) {
+		setTimeout(() => {
+			$('.add_records.js-sticky').show();
```

Comment:
Meerdere keren een element selecteren? Altijd vermijden! Maak een var aan en selecteer het element on init, waarmee je hem later kan aanroepen in resize functies. Als een gebruiker zijn browser van groot naar klein (of andersom) sleept, wordt dit 100x afgevuurd en kan dat problemen veroorzaken (performance, vertraging in resizen).